tests/functional/c/confidence_filter.py

```diff
@@ -5,7 +5,7 @@ class Client:
     """use provider class"""
 
     def __init__(self):
-        self.set_later = 0
+        self.set_later = "tomato"
```

Comment:
Reason: `E1101: Instance of 'int' has no 'lower' member (no-member)` is emitted with the changes in this PR.

Reply:
Isn't that a regression? We shouldn't emit `no-member` if we aren't certain about it's type.

Reply:
I don't think so? 
Before: `0.lower()` was not emitting a no-member. Now it is which is expected.

Reply:
But isn't `set_set_later` there so that we don't know the type of `set_later`? In this test we don't call `set_set_later` but if we were to add `self.set_set_later("")` on L9 do we still emit? If so this is a regression as then `set_later` is a `str`.

Reply:
I see your point. Thanks @DanielNoord.